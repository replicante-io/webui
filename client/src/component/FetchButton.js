'use strict';
//@flow

import { Component } from 'react';
import React from 'react';


// While refreshing: <i className="fas fa-circle-notch"></i>
class FetchButton extends React.Component<{}> {
  render() {
    return (
      <button type="button" className="btn btn-outline-secondary fetch-button">
        <i className="fas fa-redo-alt"></i>
      </button>
    );
  }
}


export default FetchButton;
