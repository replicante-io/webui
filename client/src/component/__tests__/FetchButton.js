import React from 'react';
import renderer from 'react-test-renderer';

import FetchButton from '../FetchButton';


describe('FetchButton', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<FetchButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
