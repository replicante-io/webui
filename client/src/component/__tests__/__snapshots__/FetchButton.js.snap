// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchButton renders correctly 1`] = `
<button
  className="btn btn-outline-secondary fetch-button"
  type="button"
>
  <i
    className="fas fa-redo-alt"
  />
</button>
`;
